Apps/UnitTests/Scripts/tests.js

```diff
@@ -342,25 +342,25 @@ describe("setTimeout", function () {
         setTimeout(() => {
             try {
                 expect(new Date().getTime() - startTime).to.be.at.least(10);
+                done();
             }
             catch (e) {
                 done(e);
                 return;
```

Comment:
remove this also

Reply:
I still see the extraneous `return;` statements.

Reply:
I've got this fixed locally, queued for the next push.